src/components/new-request/search.vue

```diff
@@ -32,9 +39,12 @@
               </div>
             </v-list-item-content>
 
+            <!-- render but conditionally hide disabled list items, so that the v-select
+            continues to display the current selection even when a different group is active -->
             <v-list-item-content
               v-else
               class="group-item pl-8 pr-4 py-4"
+              :class="{ 'hide-me': item.disabled }"
```

Comment:
Can't just use `d-none` because it's actually the parent element I need to hide.